packages/data-point/lib/core/transform.js

```diff
@@ -36,6 +37,7 @@ function reducerResolve (manager, reducerSource, value, options) {
   })
 
   const result = resolveFromAccumulator(manager, reducerSource, context)
+    .then(value => utils.set(context, 'value', value))
```

Comment:
many of the changes made to this file on the original PR were simplified to be the least invasive, also code had changed since the original one so some changes no longer applied